src/winetricks

```diff
@@ -18735,6 +18735,42 @@ _EOF_
 
 #----------------------------------------------------------------
 
+w_metadata hidewineexports=enable settings \
+    title="Enable hiding wine exports from Windows (wine-staging)"
```

Comment:
This should probably be something like 'Windows applications' or just 'applications'